devito/operator.py

```diff
@@ -186,6 +186,22 @@ def signature(self):
         return self.input_params + [param for param in self.output_params
                                     if param not in self.input_params]
 
+    @property
+    def total_kernel_time(self):
+        return self.propagator.timings[0]
+
+    @property
+    def total_space_loops_time(self):
+        return self.propagator.timings[1]
+
+    @property
+    def total_pre_loops_time(self):
+        return self.propagator.timings[2]
+
+    @property
+    def total_post_loops_time(self):
```

Comment:
Please confirm what I am saying but I believe pre and post loops were old names for these variables which were later changed to `time_loop_stencils`. During some rebase/merge conflict resolution these old names seem to have crept back in and right now both versions of these names exist in the code creating redundancy. AFAIK, these old names `pre_loop` and `post_loop` are not being used anywhere in the code and I have been meaning to open a PR to remove them (again). 

If you can confirm the above, please feel free to remove this.